### ISSUE TITLE
Choose Your Organization triptych not keyboard accessible

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1766]
Using keyboard navigation does not allow selection of organizations in the triptych under "Choose Your Organization." This also prevents the screenreader from reading the triptych content.

The triptych should gain focus after the top-level institution by using the tab key. Focus should also move between triptych boxes using the tab or arrow keys. The organization should be selectable using the enter or spacebar key. Once a selection has been made, the tab key should move focus to the "Start Submission" button.

**Comment by cstarcher** [comment]
Reopening as only partially fixed by #1887.

See last note here https://github.com/TexasDigitalLibrary/Vireo/pull/1887#issuecomment-1881139437.
